avaframe/com3Hybrid/hybridModel_com1DFACfg.ini

```diff
@@ -92,7 +92,7 @@ explicitFriction = 1
 frictModel = Coulomb
 #+++++++++++++General Friction parameters
 # tan of bed friction angle used for: samosAT, Coulomb, Voellmy
-mu = 0.510208802718046
+mu = 0.40375846095054385
```

Comment:
Should we limit this to f.3 or f.2?

Reply:
this is computed inside the algorithm and updated automatically...